packages/react-components/src/components/Switch/Switch.module.scss

```diff
@@ -188,4 +188,8 @@ $medium-move: $medium-width - $medium-size;
       border-width: 2px;
     }
   }
+
+  &__icon {
+    color: var(--content-black-locked);
+  }
```

Comment:
```suggestion
  &__icon svg {
    fill: var(--content-black-locked);
  }
```

Reply:
just did
```
className={styles[`${baseClass}__icon`]}
```
instead of
```
className={`${baseClass}__icon`} 
```
and it clicked :)